Linear_cell_complex/package_info/Linear_cell_complex/dependencies

```diff
@@ -23,4 +23,5 @@ Number_types
 Profiling_tools
 Property_map
 STL_Extension
+Solver_interface
```

Comment:
`Solver_interface` is LGPL, so that addition is fine.

However, it is strange. Which new `#include` triggered that change?